libs/designsystem/src/lib/components/card/directives/card-is-clickable.directive.ts

```diff
@@ -0,0 +1,16 @@
+import { Directive, OnInit } from '@angular/core';
+
+import { CardComponent } from '../card.component';
+
+@Directive({
+  // tslint:disable-next-line
+  selector: 'kirby-card[click]',
+})
+export class CardIsClickableDirective implements OnInit {
+  constructor(private card: CardComponent) {}
+
+  ngOnInit(): void {
+    this.card.cardIsClickable = true;
```

Comment:
Consider removing this line based on above details.

Reply:
For the above to work, the card needs to be focusable by the keyboard. Consider adding another hostbinding for this, like 
`  @HostBinding('attr.tabindex') tabindex: number = 0;`